Source/Libraries/Adapters/MetadataAdapters/MetadataImportAdapter.cs

```diff
@@ -911,7 +911,7 @@ private Action<DataSet> GetSynchronizeMetadataAction()
                                 List<int> sourceIndicies;
 
                                 if (definedSourceIndicies.TryGetValue(id, out sourceIndicies))
-                                    command.ExecuteNonQuery(deletePhasorSql + $" AND SourceIndex NOT IN ({string.Join(",", sourceIndicies)})", MetadataSynchronizationTimeout, id);
+                                    command.ExecuteNonQuery(deletePhasorSql + " AND SourceIndex NOT IN ({0})", string.Join(",", sourceIndicies), MetadataSynchronizationTimeout, id);
```

Comment:
this won't work either because it turns a list of `[1,2,3]` into a string `"1,2,3"` where `1` is not in `("1,2,3")`

Reply:
updated